arangodb-net-standard.Test/DatabaseApi/DatabaseApiClientTestFixture.cs

```diff
@@ -24,6 +24,8 @@ public class DatabaseApiClientTestFixture : ApiClientTestFixtureBase
         /// </summary>
         public DatabaseApiClient DatabaseClientOther { get; internal set; }
 
+        public string DeletableDatabase { get; } = "DeletableDatabase";
```

Comment:
Should we use a more specific database name? People could be running these tests locally on their machine. There is less risk of mismatch if we use the class name, e.g. `DatabaseApiClientTestFixture` + "_ToBeDeleted" or something.